SpeechService/SpeechService.cs

```diff
@@ -18,14 +19,31 @@
 namespace EddiSpeechService
 {
     /// <summary>Provide speech services with a varying amount of alterations to the voice</summary>
-    public class SpeechService
+    public class SpeechService : INotifyPropertyChanged
     {
         private SpeechServiceConfiguration configuration;
 
         private static readonly object activeSpeechLock = new object();
         private ISoundOut activeSpeech;
         private int activeSpeechPriority;
 
+        private static bool? _eddiSpeaking;
+        public static bool? eddiSpeaking
```

Comment:
Not sure why this is optional? My rule of thumb is only make a property optional if there's no sensible default value, but in this case a default of `false` seems sensible unless I'm missing something.

Keeping properties non-optional when possible tends to simplify any code that uses the class.